packages/preview/tuhi-booklet-vuw/0.1.0/tuhi-booklet-vuw.typ

```diff
@@ -0,0 +1,173 @@
+
+#let smcps = (x) => text(tracking: 0.8pt)[#smallcaps[#lower(x)]]
+
+#let sentence_case(x) = text[#upper(x.at(0))#x.slice(1)]
```

Comment:
Some other functions in this package use kebab-case, and following that convention what is considered to be the good practice for Typst packages, could you please rename this function and the ones below? Or if you really prefer snake_case, rename the ones that use kebab-case so that the package only uses one convention.

Reply:
I've converted them to kebab-case, thanks